### PR TITLE
Removed extra file.

### PR DESCRIPTION
**Pull request by hejazizo**
Added

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,9 +0,0 @@
-print('hello world')
-
-def func(a, b):
-    return a+b
-
-print()
```
Comment (hejazizo) [inline main.py]: in print chie zadi?
masalan barnamenevisi!

**Comment by zdaneshz** [comment]
These changes does not make sense

File: main.py (hunk 1 above)
Comment (SinaDibaji) [inline main.py]: Good job!

**Comment by zdaneshz** [comment]
WTH!

File: main.py (hunk 1 above)
Comment (hejazizo) [reply main.py]: thank you!

**Pull request merged**